assets/queries/k8s/pod_network_policy_not_configured/query.rego

```diff
@@ -4,58 +4,50 @@ CxPolicy[result] {
 	document := input.document[i]
 	metadata := document.metadata
 
-	object.get(document, "kind", "undefined") != "NetworkPolicy"
+	document.kind == "NetworkPolicy"
+	spec := document.spec
 
-	findNetworkPolicy(document.metadata.labels[key], key) == false
+	not findPod(true)
 
-	result := {
-		"documentId": input.document[i].id,
-		"searchKey": sprintf("metadata.name=%s.labels.%s", [metadata.name, key]),
+    result := {
+		"documentId": document.id,
+		"searchKey": sprintf("metadata.name=%s", [metadata.name]),
 		"issueType": "MissingAttribute",
-		"keyExpectedValue": sprintf("'metadata.labels.%s' is targeted by a network policy", [key]),
-		"keyActualValue": sprintf("'metadata.labels.%s' is not targeted by a network policy", [key]),
+		"keyExpectedValue": "Pod is defined",
+		"keyActualValue": sprintf("metadata.name=%s should target at least one Pod", [metadata.name])
```

Comment:
The ActualValue mentions what values the resource has that are faulty, so the message should reflect that, and not what there should be